code/implementation/extension-api/src/main/resources/META-INF/cattle/extension-api/spring-extension-api-context.xml

```diff
@@ -17,6 +17,7 @@
                 <value>io.cattle.platform.extension.api.model.ResourceDefinition</value>
                 <value>io.cattle.platform.engine.process.StateTransition</value>
                 <value>io.cattle.platform.iaas.api.auth.github.Token</value>
+                <value>io.cattle.platform.iaas.api.auth.config.GithubConfig</value>
```

Comment:
Wrong xml file.  This should go in a spring XML for iaas-api-logic.  Same with the `io.cattle.platform.iaas.api.auth.github.Token`